### PR TITLE
Fix typos 

### PR DESCRIPTION
**Pull request by ml0mbardi**
- Align kwargs mapping to `pixle_attention_mask` so the mask is propagated (previous `pixel_attention_mask` mismatch dropped it as None).

- Fix small typos in comments:
  bandwith (floa32) → bandwidth (float32)
  overide  → override
  appling  → applying 
  Recursivley → Recursively

### CODE REVIEW

File: optimum/intel/openvino/quantization.py [hunk 1]
```diff
@@ -1449,6 +1449,121 @@ def _quantize_ovbasemodel(
...
             ov_config.save_pretrained(save_directory)
 
+    def _quantize_torchmodel(
+        self,
+        ov_config: OVConfig,
+        save_directory: Union[str, Path],
+        calibration_datasets: Optional[OVCalibrationDataset] = None,
+        file_name: Optional[str] = None,
+        **kwargs,
+    ):
```
Comment (nikita-savelyevv) [inline optimum/intel/openvino/quantization.py]: I believe this is a merge artifact because this logic has been removed recently. Could you please go over the changes and check if they really belong to the PR?

File: optimum/exporters/openvino/model_patcher.py [hunk 2]
```diff
@@ -77,6 +77,78 @@ def patch_update_causal_mask(
             inner_model._update_causal_mask = types.MethodType(patch_fn, inner_model)
 
 
+# adopted from
```
Comment (rkazants) [inline optimum/exporters/openvino/model_patcher.py]: please remove this big addition. Check below as well

Comment (ml0mbardi) [reply optimum/exporters/openvino/model_patcher.py]: @ rkazants
Hi! All cleaned up! Just typo fixes now.

File: optimum/intel/openvino/quantization.py (hunk 1 above)
Comment (ml0mbardi) [reply optimum/intel/openvino/quantization.py]: @ nikita-savelyevv
Hi! Rebased and cleaned up already—merge artifacts are gone now. Thanks!

File: optimum/intel/openvino/modeling_visual_language.py [hunk 3]
```diff
@@ -916,7 +916,7 @@ def prepare_inputs_for_generation(
                 "image_grid_thw": kwargs.get("image_grid_thw"),
                 "video_grid_thw": kwargs.get("video_grid_thw"),
                 "token_type_ids": kwargs.get("token_type_ids"),
-                "pixel_attetion_mask": kwargs.get("pixle_attetion_mask"),
+                "pixle_attention_mask": kwargs.get("pixle_attention_mask"),
```
Comment (nikita-savelyevv) [inline optimum/intel/openvino/modeling_visual_language.py]:
```suggestion
                "pixel_attention_mask": kwargs.get("pixel_attention_mask"),
```
